burn-import/onnx-tests/tests/tanh/tanh.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+# used to generate model: onnx-tests/tests/tanh/tanh.onnx
+
+import torch
+import torch.nn as nn
+
+
+class Model(nn.Module):
+    def __init__(self):
+        super(Model, self).__init__()
+
+    def forward(self, x):
+        return torch.tanh(x)
+
+
+def main():
+
```

Comment:
Can we add rand.seed for reproducibility like this:

```python
    # Set random seed for reproducibility
    torch.manual_seed(0)
```

Reply:
Done with it.